### PR TITLE
Events Near Me for Users

### LINKED ISSUE #85: events near me

### PR DESCRIPTION
**Pull request by shweta-kumaran**
Issue:  https://github.com/users/shweta-kumaran/projects/2/views/1?pane=issue&itemId=86133834&issue=shweta-kumaran%7CCastNXTfall24%7C85 

Users will be able to see events that are near their city and then state.  This will allow users to see opportunities near them easily. This is based on the most recent talent data that the user puts down in regards to city and state. 

Added tests for the code but noted that main had low coverage. So added random tests to get coverage to 90.28%

### CODE REVIEW

File: app/controllers/user_controller.rb [hunk 1]
```diff
@@ -51,9 +56,19 @@ def index
         if "ACCEPTING".casecmp? event.status
           acceptingTableData << object
         end
+        if "ACCEPTING".casecmp?(event.status) && event.statename == user_state
```
Comment (rahulb99) [inline app/controllers/user_controller.rb:59]: duplicate if condition. could add as nested condition

File: spec/javascript/Components/Client/ClientEventFeedback.spec.js [hunk 2]
```diff
@@ -180,3 +180,59 @@ test('open the chat window', () => {
...
+    const properties = {
+        ...PROPERTIES_CLIENT_FEEDBACK,
+        data: { announcements: [{ announcementContent: "Event Update", timeSent: new Date() }] }
+    };
+    const view = ReactTestUtils.renderIntoDocument(<ClientEventFeedback properties={properties} />);
+    const announcementList = view.state.announcements;
+    expect(announcementList.length).toBeGreaterThan(0);
+});
+
+test('chat window toggle and messaging', async () => {
```
Comment (rahulb99) [inline spec/javascript/Components/Client/ClientEventFeedback.spec.js:193]: 👍🏼

**Review by rahulb99 — APPROVED** [review]
LGTM

File: app/controllers/user_controller.rb (hunk 1 above)
Comment (shweta-kumaran) [reply app/controllers/user_controller.rb:59]: It is not duplicated as it is two different lists that are being created. The first if checks if valid for the general list of events and the second generated reccomendations

**Pull request merged**